common/config/azure-pipelines/jobs/docs-ci.yaml

```diff
@@ -0,0 +1,50 @@
+# iTwin.js Docs CI Build
+
+trigger:
+  branches:
+    include:
+      - master
+      - release/*
+    exclude:
+      - release/2*
+      - release/1*
+
+pr:
+  drafts: false
+  branches:
+    include:
+      - master
+      - release/*
+    exclude:
+      - release/2*
+      - release/1*
+
+jobs:
+  - job:
+    workspace:
+      clean: all
+    pool:
+      name: iModelTechCI
+      demands: Agent.OS -equals Windows_NT
+
+    steps:
+      - checkout: self
+        clean: true
+
+      - task: NodeTool@0
+        displayName: Use Node 14
+        inputs:
+          versionSpec: 14.17.4
+          checkLatest: true
+
+      - script: |
+          git config --local user.email imodeljs-admin@users.noreply.github.com
+          git config --local user.name imodeljs-admin
+        displayName: Setup git config
+
+      - template: ../templates/docs-build.yaml
+
+      - publish: $(Agent.BuildDirectory)/tempDocsBuild/
```

Comment:
Is this enough to publish? I think the release pipeline expects a nuget package

Reply:
I've moved cred scanning and nuget packaging from the docs CI build into the the release pipeline. I've added a duplicate series of deployments steps postfixed with "YAML test" for testing with this new artifact, you should be able to see logs in [this release](https://dev.azure.com/bentleycs/beconnect/_releaseProgress?_a=release-pipeline-progress&releaseId=166347).